cmd/kubermatic-operator/main.go

```diff
@@ -90,12 +90,11 @@ func main() {
 	v := common.NewDefaultVersions()
 	log.With("kubermatic", v.Kubermatic, "ui", v.UI).Infof("Moin, moin, I'm the Kubermatic %s Operator and these are the versions I work with.", v.KubermaticEdition)
 
-	config, err := clientcmd.BuildConfigFromFlags("", opt.kubeconfig)
-	if err != nil {
-		log.Fatalw("Failed to build config", zap.Error(err))
-	}
-
-	mgr, err := manager.New(config, manager.Options{MetricsBindAddress: opt.internalAddr})
+	mgr, err := manager.New(ctrl.GetConfigOrDie(), manager.Options{
+		MetricsBindAddress: opt.internalAddr,
+		LeaderElection:     opt.enableLeaderElection,
+		LeaderElectionID:   "b46c853f.kubermatic.io",
```

Comment:
Where is this ID coming from?

Reply:
this is just a randomly generated hex word + `.kubermatic.io`

Reply:
I see. Can we change it to `uz32g423zrt.kubermatic.io`? I like my randomness better.

No but seriously, this feels weird. It looks to me like "Oh, that's probably [clusterID].kubermatic.io", even though it has nothing to do with that. Why does this need to be random? Couldn't this be "operator.kubermatic.io"? I assume this is namespaced, right? Or are we doing leaderelection in the kube-system namespace?

Reply:
yes it can be changed to `operator.kubermatic.io` without any problem, I think I copied this from a skeleton generated with kubebuilder. I'll change this. The namespace if not specified is the one the operator is running into.